src/doc/rustdoc/src/unstable-features.md

```diff
@@ -100,28 +100,16 @@ Note: Because of how `macro_rules` macros are scoped in Rust, the intra-doc link
 These features operate by extending the `#[doc]` attribute, and thus can be caught by the compiler
 and enabled with a `#![feature(...)]` attribute in your crate.
 
-### Documenting platform-/feature-specific information
+### `#[doc(cfg)]`: Telling rustdoc what platforms documentation is present on
```

Comment:
It's not just for platforms, it's what cargo features need to be enabled for example.

This line conflicts with #76849 so maybe #76849 should be closed in favor of this?